### PR TITLE
Handle svg hero images in blocks/cover shortcode

### PR DESCRIPTION
**Pull request by fekete-robert**

### CODE REVIEW

**Review by chalin — COMMENTED** [review]
Seems reasonable to me. 
WDYT @deining @geriom?

File: layouts/shortcodes/blocks/cover.html [hunk 1]
```diff
@@ -8,9 +8,13 @@
...
-{{ with $promo_image -}}
-{{ $promo_image_big := (.Fill (printf "1920x1080 %s" $image_anchor)) -}}
-{{ $promo_image_small := (.Fill (printf "960x540 %s" $image_anchor)) -}}
+{{ with $promo_image }}
+{{ $promo_image_big := . }}
+{{ $promo_image_small := . }}
+{{ if ne $promo_image.MediaType.SubType "svg" }}
+  {{ $promo_image_big = (.Fill (printf "1920x1080 %s" $image_anchor)) }}
+  {{ $promo_image_small = (.Fill (printf "960x540 %s" $image_anchor)) }}
+{{ end }}
```
Comment (chalin) [inline layouts/shortcodes/blocks/cover.html:17]:
```suggestion
{{ with $promo_image -}}
{{ $promo_image_big := . -}}
{{ $promo_image_small := . -}}
{{ if ne $promo_image.MediaType.SubType "svg" -}}
  {{ $promo_image_big = .Fill (printf "1920x1080 %s" $image_anchor) -}}
  {{ $promo_image_small = .Fill (printf "960x540 %s" $image_anchor) -}}
{{ end -}}
```

**Review by chalin — APPROVED** [review]
Whitespace clean up, otherwise LGTM

**Comment by fekete-robert** [comment]
Thanks for the review, @chalin !

**Review by chalin — APPROVED** [review]
Thanks for the PR @fekete-robert

**Pull request merged**